system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -414,7 +414,7 @@
         ["Collection with Entry Title [correct], Short Name [S2], Version Id [V1] referenced in granule [Gran1] provider [PROV1] does not exist."]))))
 
 (deftest ingest-granule-with-parent-umm-collection-test
-  (let [cddis-umm (-> "example_data/umm-json/1.2/CDDIS.json" io/resource slurp)
+  (let [cddis-umm (-> "example-data/umm-json/collection/v1.2/CDDIS.json" io/resource slurp)
```

Comment:
CMR naming convention has directory names using underscores, not dashes. Not sure why you change it.

Reply:
Directories for Clojure namespaces are required to have underscores, but this is not source code. The dash is actually more prevalent for non-source code CMR dirs than underscores, especially for higher-level directories.

Reply:
All dirs:
```
$ find . -type d|sort -u|egrep -v 'src|test'|grep '-'|wc -l
    1741
$ find . -type d|sort -u|egrep -v 'src|test'|grep '_'|wc -l
     596
```

Higher-level dirs:

```
$ find . -type d -maxdepth 3|sort -u|egrep -v 'src|test'|grep '-'|wc -l
     202
$ find . -type d -maxdepth 3|sort -u|egrep -v 'src|test'|grep '_'|wc -l
       9
```

Reply:
Since we aren't consistent I'd like to change to use spaces. 😛

Reply:
Man, I guess nobody paid any attention. It stands out when you actually renamed it to something else. I don't care then. :)